src/main/kotlin/app/revanced/patches/youtube/video/videoqualitymenu/RestoreOldVideoQualityMenuPatch.kt

```diff
@@ -76,6 +81,28 @@ object RestoreOldVideoQualityMenuPatch : BytecodePatch(
             }
         }
 
+        // Force YT to add the 'advanced' quality menu for Shorts.
+        VideoQualityMenuOptionsFingerprint.resultOrThrow().let {
+            val result = it.scanResult.patternScanResult!!
+            val startIndex = result.startIndex
+            val endIndex = result.endIndex
+
+            it.mutableMethod.apply {
+                val freeRegister = getInstruction<OneRegisterInstruction>(startIndex).registerA
+
+                // Add a conditional branch around the 3 menu code to force the 4 menu (Advanced menu) code path.
```

Comment:
Our current patch simulates a click, probably figuring out the handler involved in the future to invoke would be more durable. For now:

```suggestion
                // A condition controls whether to show the three or four items quality menu.
                // Force the four items quality menu to make the "Advanced" item visible, necessary for the patch.
```